desktop/src/main/java/bisq/desktop/main/support/dispute/agent/SignedOfferView.java

```diff
@@ -0,0 +1,471 @@
+
+import javafx.fxml.FXML;
+
+import javafx.scene.control.ContextMenu;
+import javafx.scene.control.Label;
+import javafx.scene.control.MenuItem;
+import javafx.scene.control.TableCell;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableRow;
+import javafx.scene.control.TableView;
+import javafx.scene.control.Tooltip;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Priority;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+
+import javafx.geometry.Insets;
+
+import javafx.beans.property.ReadOnlyObjectWrapper;
+
+import javafx.collections.ListChangeListener;
+import javafx.collections.transformation.SortedList;
+
+import javafx.util.Callback;
+import javafx.util.Duration;
+
+import java.util.Comparator;
+import java.util.Date;
+
+@FxmlView
+public class SignedOfferView extends ActivatableView<VBox, Void> {
+
+    private final OpenOfferManager openOfferManager;
+
+    @FXML
+    protected TableView<SignedOffer> tableView;
+    @FXML
+    TableColumn<SignedOffer, SignedOffer> dateColumn;
+    @FXML
+    TableColumn<SignedOffer, SignedOffer> offerIdColumn;
+    @FXML
+    TableColumn<SignedOffer, SignedOffer> reserveTxHashColumn;
+    @FXML
+    TableColumn<SignedOffer, SignedOffer> reserveTxHexColumn;
+    @FXML
+    TableColumn<SignedOffer, SignedOffer> reserveTxKeyImages;
+    @FXML
+    TableColumn<SignedOffer, SignedOffer> arbitratorSignatureColumn;
+    @FXML
+    TableColumn<SignedOffer, SignedOffer> reserveTxMinerFeeColumn;
+    @FXML
+    TableColumn<SignedOffer, SignedOffer> makerTradeFeeColumn;
+    @FXML
+    InputTextField filterTextField;
+    @FXML
+    Label numItems;
+    @FXML
+    Region footerSpacer;
+
+    private SignedOffer selectedSignedOffer;
+
+    private XmrWalletService xmrWalletService;
+
+    private TradeManager tradeManager;
+
+    private ContextMenu contextMenu;
+
+    private OfferDetailsWindow offerDetailsWindow;
+
+    private final ListChangeListener<SignedOffer> signedOfferListChangeListener;
+
+    @Inject
+    public SignedOfferView(OpenOfferManager openOfferManager, XmrWalletService xmrWalletService,
+                           TradeManager tradeManager, OfferDetailsWindow offerDetailsWindow) {
+        this.openOfferManager = openOfferManager;
+        this.xmrWalletService = xmrWalletService;
+        this.tradeManager = tradeManager;
+        this.offerDetailsWindow = offerDetailsWindow;
+
+        signedOfferListChangeListener = change -> applyList();
+    }
+
+    private void applyList() {
+        SortedList<SignedOffer> sortedList = new SortedList<>(openOfferManager.getObservableSignedOffersList());
+        sortedList.comparatorProperty().bind(tableView.comparatorProperty());
+        tableView.setItems(sortedList);
+
+        numItems.setText(Res.get("shared.numItemsLabel", sortedList.size()));
+    }
+
+    private Offer fetchOffer(SignedOffer signedOffer) {
+        Trade trade = tradeManager.getTrade(signedOffer.getOfferId());
```

Comment:
This should not be fetching the trade. It should only use fields available in SignedOffer.

Reply:
OK. Removed.